src/puppeteer_pool.js

```diff
@@ -217,6 +217,7 @@ class PuppeteerPool {
             }
 
             const browser = await launchPuppeteerFunction(opts);
+            // Maybe apply hiding tricks
```

Comment:
I'd keep this functionality just in `Apify.launchPuppeteer()`. If the user doesn't use it in provided `launchPuppeteerFunction` then he has to configure this manually...